var/spack/repos/builtin/packages/qthreads/package.py

```diff
@@ -85,3 +85,58 @@ def configure_args(self):
                     % self.spec.variants['scheduler'].value)
 
         return args
+
+
+
+    test_base_path = 'test/basics/'
```

Comment:
For readability, the three properties -- `test_requires_compiler` *and* the two here -- should be moved to under the `url` property at the top of the package.